concepts/categories.md

```diff
@@ -1,131 +0,0 @@
----
-layout: documentation
-title: Categories
----
-
-# Categories
-
-Categories in openHAB are used to provide meta information about Things, Channels, etc. UIs can use this information to render specific icons or provide a search functionality to for example filter all Things for a certain category.
-
-## Differences between categories
-
-We separate the categories into `functional` and `visual`.
-Therefore we treat `Thing categories` as how the physical device **looks like** and `Channel categories` as something that describes the **functional purpose** of the Channel.
-
-## Thing Categories
-
-The Thing type definition allows to specify a category.
```

Comment:
I have noticed that this part of the page is actually referenced by the developer Thing XML docs:
https://github.com/search?q=repo%3Aopenhab%2Fopenhab-docs%20categories.html&type=code

I am not sure how to process, but AFAIK the available categories are basically the list of icons with the icon names in the typical Java class naming style (e.g. FrontDoor instead of lowercase icon name frontdoor).
The category of a Thing and channels is AFAIK only setting the icon and not doing anything else.

Therefore I propose to remove the references to this page, and instead write under the Thing categories and the channel categories heading something like:

> Thing [channel] categories are used to set an icon for that Thing [channel]. The available categories correspont with the available icons of the [classic icon set]({{base}}/configuration/iconsets/classic/), however categories are written in Java class-naming style, e.g. `FrontDoor` instead of lowercase `frontdoor`.

Reply:
I do not see how can Channels or Things get an icon/a category in file-based definitions.  Is it possible to set categories for Things and Channels only at development time, and these are then fixed forerer in the `.jar` file?

Reply:
Exactly, these are defined by the add-on developer inside the thing-type and channel-type XML definitions and used by the UI as suggested icons when you „create Items from equipment“.